Dockerfile-deploy

```diff
@@ -6,6 +6,12 @@ MAINTAINER info@codegram.com
 ENV RAILS_ENV=production
 ENV PORT=3000
 
+RUN cd /tmp && \
+    decidim decidim_app --skip-bundle && \
+    cd decidim_app && \
+    bundle install --without development,test && \
```

Comment:
Missing `npm install`?

Reply:
Node dependencies are not installed globally, so there's nothing to cache here. Which makes me think that maybe we should be installing the npm dependencies globally as well.

Reply:
Is `npm` needed in production apps at all? As far as I understand our current situation, `npm` is only a development tool to be able to generate the `bundle.js` asset?

Reply:
Exactly, it isn't needed anyways 👍

Reply:
Cool, thanks!!